src/layout/index.ts

```diff
@@ -1,19 +1,17 @@
 import * as boxmodel from "@src/layout/boxmodel";
-import * as discover from "@src/layout/discover";
 import * as doc from "@src/layout/document";
 import * as dom from "@src/layout/dom";
 import * as mutation from "@src/layout/mutation";
 
 export function start(): void {
     doc.reset();
-    dom.reset();
+    dom.start();
     mutation.start();
-    discover.start();
```

Comment:
What's the semantics of star/tend/reset and why aren't they all the same here in the start/end methods?

Reply:
I'll come back to it later. In some cases there's no need to do anything in the end function so i have left it out, instead of adding bogus bytes. I will reconsider this later and see if it makes sense to reintroduce for consistency anyway.